testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -75,7 +75,7 @@ EOF
 )"
   fi
   python -m jenkins.pull_request_creator --repo-root="$repo_dir" \
-         --base-branch-name="cleanup-python-code" --commit-message="$message" \
```

Comment:
Is this a decision we can push back up to the caller?

Reply:
+1 this. Seems like a good default, but could the script be used for something else besides a feat?

Reply:
We can always make another optional file

Reply:
Oh, I just meant that the caller would include the conventional-commit prefix in their commit message from the get-go.

Reply:
Still a little bit confused. Do you mean as part of the job config?